lms/djangoapps/organizations/views.py

```diff
@@ -18,25 +18,35 @@
 from organizations.models import Organization, OrganizationGroupUser
 from api_manager.users.serializers import SimpleUserSerializer
 from api_manager.groups.serializers import GroupSerializer
-from api_manager.permissions import SecureListAPIView
+from api_manager.permissions import SecureListAPIView, APIModelViewSet
 from api_manager.utils import str2bool
 from gradebook.models import StudentGradebook
 from student.models import CourseEnrollment
 from student.roles import get_aggregate_exclusion_user_ids
 
-from .serializers import OrganizationSerializer, BasicOrganizationSerializer
+from .serializers import OrganizationSerializer, BasicOrganizationSerializer, OrganizationWithCourseCountSerializer
 
 
-class OrganizationsViewSet(viewsets.ModelViewSet):
+class OrganizationsViewSet(APIModelViewSet):
     """
     Django Rest Framework ViewSet for the Organization model.
     """
     serializer_class = OrganizationSerializer
     model = Organization
 
     def list(self, request, *args, **kwargs):
-        self.serializer_class = BasicOrganizationSerializer
-        return super(OrganizationsViewSet, self).list(request, *args, **kwargs)
+        self.serializer_class = OrganizationWithCourseCountSerializer
+        queryset = self.get_queryset()
+        queryset = queryset.annotate(
+            number_of_courses=Count('groups__coursegrouprelationship__course_id', distinct=True)
+        )
+        page = self.paginate_queryset(queryset)
```

Comment:
I think we should move this logic inside `list` method of `APIModelViewSet ` we created and just call it here like this 
`return super(OrganizationsViewSet, self).list(request, *args, **kwargs)`